packages/chrome/src/utils/useSheetContext.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { createContext, useContext } from 'react';
+
+export interface ISheetContext {
+  titleId: string;
+  descriptionId: string;
+}
+
+export const SheetContext = createContext<ISheetContext>({
+  titleId: 'sheet--title',
+  descriptionId: 'sheet--description'
```

Comment:
Is there a reason these are being initialized here? These would almost never be valid ID values.

Reply:
Good call on this, will remove.